components/server/ee/src/prebuilds/bitbucket-server-service.ts

```diff
@@ -48,16 +48,15 @@ export class BitbucketServerService extends RepositoryService {
 
         const identity = user.identities.find((i) => i.authProviderId === this.authProviderConfig.id);
         if (!identity) {
-            console.error(
-                `Unexpected call of canInstallAutomatedPrebuilds. Not authorized with ${this.authProviderConfig.host}.`,
-            );
+            console.error(`BBS: not authorized.`, { host: this.authProviderConfig.host });
```

Comment:
Isn't this a different error? Here, we just don't have any identity configured, not that you're not authorized.

Reply:
Good question. If no token exists for a host to talk to, would that be an "unauthorized" case? We used to call/treat it like that. 

Currently the relation is model like this `User -> Identity -> Token`, hence if no identity is present there can't be a token for this host. 

I'll change to something like "no identity found".  Ok?